mmpose/core/evaluation/top_down_eval.py

```diff
@@ -349,6 +427,8 @@ def keypoints_from_heatmaps(heatmaps,
         assert post_process not in [False, None, 'megvii']
     if post_process in ['megvii', 'unbiased']:
         assert kernel > 0
+    if use_udp:
```

Comment:
Looking at the change, I'm wandering if we can add an `udp` option in the `post_process` argument? What do you think?

Reply:
emmm, i think it is unnecessary, as i have remove the concept of 'post_process' in udp~  i add an assertion here to prevent this:
>if post_process == 'megvii':
>        maxvals = maxvals / 255.0 + 0.5